src/clientdlg.cpp

```diff
@@ -1027,6 +1031,34 @@ void CClientDlg::OnTimerSigMet()
     // show current level
     lbrInputLevelL->SetValue ( pClient->GetLevelForMeterdBLeft() );
     lbrInputLevelR->SetValue ( pClient->GetLevelForMeterdBRight() );
+
+    if ( bDetectFeedback &&
+        ( pClient->GetLevelForMeterdBLeft() > NUM_STEPS_LED_BAR - 0.5  ||
+        pClient->GetLevelForMeterdBRight() > NUM_STEPS_LED_BAR - 0.5 ) )
+    {
+        // mute locally and mute channel
+        chbLocalMute->setCheckState ( Qt::Checked );
+        MainMixerBoard->MuteMyChannel();
+
+        // show message box about feedback issue
+        QCheckBox* chb = new QCheckBox ( tr ( "Enable feedback detection" ) );
+        chb->setCheckState ( pSettings->bEnableFeedbackDetection ?
+            Qt::Checked : Qt::Unchecked );
+        QMessageBox msgbox;
+        msgbox.setText ( tr ( "Audio feedback or loud signal detected.\n\n"
+            "We muted your channel and activated 'Mute Myself'. Please "
+            "solve the feedback issue first and unmute yourself afterwards. "
+            "Did you plug-in earphones?") );
```

Comment:
> Did you plug-in earphones?

I'm not sure this would be helpful.  Certainly with ASIO4ALL, it could have all outputs enabled so plugging in _head_phones wouldn't prevent feedback from built-in speakers, necessarily.  I've had to tell people to take off their headphones and play so that they believe that their speakers are still sounding...

Reply:
In fact, I'd rather it pointed to the "Hints and Tips" page on the web site and _that_ actually had a "methods for resolving feedback" section.

Reply:
Ok, I can follow this reasoning and have removed the hint. Once the website provides a section about resolving feedback, we could add a corresponding link.